docs/src/dedicated-gen-2/overview/backups.md

```diff
@@ -2,23 +2,17 @@
 title: "Backups"
 weight: 4
 toc: false
-description: |
-  Platform.sh takes a byte-for-byte snapshot of {{% names/dedicated-gen-2 %}} production environments every six (6) hours. They're retained on a sliding scale, so more recent time frames have more frequent backups.
+description: See when backups of {{% names/dedicated-gen-2 %}} environments are taken
```

Comment:
@CollierCZ ❓ More of a Github question than anything, but I'm confused as to why this suggestion would be displayed in the Conversation tab but not in the Files changed tab. I have noticed similar instances before and I don't really understand it as it doesn't show as Outdated or anything. Is it due to something I did when submitting it?

Reply:
Interesting! I didn't notice, since I was just looking t the Files changed tab. I'm not sure how you added the comment, but it might affect it? Or maybe if you added it at a different time? It seems like all part of the same review, so I don't really know. Maybe just a bug?

Reply:
Strange. I don't think I did anything different but I'll pay attention next time.